media/css/firefox/whatsnew/whatsnew-115-eu-mobile.scss

```diff
@@ -0,0 +1,72 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/protocol/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import 'includes/base';
+@import 'includes/dark-mode';
+
+// * -------------------------------------------------------------------------- */
+// Main content
+
+.wnp-content-main {
+    color: $color-black;
+}
+
+.wnp-main-image {
+    margin: 0 auto;
+    max-width: $content-md;
+}
+
+.wnp-main-title {
+    @include text-title-lg;
+
+    @media #{$mq-lg} {
+        @include text-title-xl;
+    }
+}
+
+.wnp-main-tagline {
+    @include text-body-lg;
+}
+
+.wnp-main-cta {
+    margin-top: $spacing-2xl;
+}
+
+.wnp-content-extra {
+    margin-top: $layout-xl;
+}
+
+.c-qrcode {
+    background: $color-white;
+    border-radius: $border-radius-md;
+    border: 3px solid $color-black;
+    padding: $spacing-xl $spacing-sm;
```

Comment:
The side padding here seems a little too tight, though only at certain in-between sizes before text wraps. Maybe bump it up to `$spacing-md`?

![image](https://github.com/mozilla/bedrock/assets/205591/11c232bf-6c47-4b75-af6a-a54524b77beb)